test/test.js

```diff
@@ -136,5 +154,39 @@ describe('test', function () {
         });
       });
     });
-  }
+  });
+
+  describe.only('#kill() timeout: ', function () {
+    it('it should timeout after 30secs by default if the killing is not successful', function(done) {
+      mockKill();
+      var killStartDate = Date.now();
+      PS.lookup({pid: pid}, function (err, list) {
+        assert.equal(list.length, 1);
+        PS.kill(pid, function (err) {
+          assert.equal(Date.now() - killStartDate >= 30 * 1000, true);
+          assert.equal(err.message.indexOf('timeout') >= 0, true);
+          restoreKill();
+          PS.kill(pid, function(){
+            done();
+          });
+        });
+      });
+    });
```

Comment:
It's a little annoying for the test to have to wait the default 30 seconds. I get why you want to cover this case though. I would have been okay with a 5 or 10 second timeout to be honest.